pkg/node/node.go

```diff
@@ -239,6 +227,38 @@ func NewBee(o Options) (*Bee, error) {
 		b.debugAPIServer = debugAPIServer
 	}
 
+	var wg sync.WaitGroup
+
+	// TODO: be more resilient on connection errors and connect in parallel
+	for _, a := range o.Bootnodes {
+		wg.Add(1)
+		go func(aa string) {
+			defer wg.Done()
+			addr, err := ma.NewMultiaddr(aa)
+			if err != nil {
+				logger.Debugf("multiaddress fail %s: %w", aa, err)
```

Comment:
I think that verb `%w` is not well interpreted in logger. Should be `%v`.

This goes to other lines below.